cmd/hummingbird/main.go

```diff
@@ -499,6 +509,9 @@ func main() {
 	case "andrewd":
 		andrewdFlags.Parse(flag.Args()[1:])
 		srv.RunDaemon(tools.NewAdmin, andrewdFlags)
+	case "object-info":
```

Comment:
can we change this to `oinfo` or something shorter? not a huge deal

Reply:
oinfo it is